src/tests/ftest/vmd/fault_reintegration.py

```diff
@@ -0,0 +1,255 @@
+            4. While IOR (EC+GX) in progress put single drive to fault mode.
+            5. IOR will halt until rebuild is finished.
+            6. IOR should complete without any error.
+            7. Drive status LED should be solid Amber ON indicating it’s faulty.
+            8. Verify IOR data and check container.
+            9. Replace the same drive back.
+            10. Drive status LED should be off indicating good device is plugged-in.
+        Args:
+            oclass (str, optional): object class (eg: RP_2G8, S1,etc). Defaults to None
+        """
+        # 1.
+        self.log_step("Bring up server and reset led of all devices.")
+        if oclass is None:
+            oclass = self.ior_cmd.dfs_oclass.value
+        device_info = get_storage_query_device_info(self, self.dmg)
+        devices = [device['uuid'] for device in device_info]
+        self.log.info("Device information")
+        self.log.info("==================")
+        for index, entry in enumerate(device_info):
+            self.log.info("Device %s:", index)
+            for key in sorted(entry):
+                self.log.info("  %s: %s", key, entry[key])
+        test_dev = devices[0]
+        for device in devices:
+            get_dmg_response(self, self.dmg.storage_led_identify, reset=True, ids=device)
+        # 2.
+        self.log_step("Verify that each device is in a 'NORMAL' state and its LED is 'OFF'")
+        err_dev = []
+        for device in devices:
+            if not self.verify_dev_led_state(device, 'NORMAL', 'OFF'):
+                err_dev.append(device)
+        if err_dev:
+            self.fail("#Device {} not in expected NORMAL, OFF state".format(err_dev))
+
+        # 3.
+        self.log_step("Creating a pool and container with RF and starting IOR in a thread")
+        self.add_pool(connect=False)
+        self.add_container(self.pool)
+
+        job_manager = get_job_manager(self, subprocess=None, timeout=120)
+        thread_queue = Queue()
+
+        ior_test_seq = self.params.get("ior_test_sequence", "/run/ior/iorflags/*")[0]
+        thread = []
+        kwargs = {"thread_queue": thread_queue, "job_id": 1}
+        ior_kwargs = {
+            "test": self,
+            "manager": job_manager,
+            "log": "ior_thread_write_pool_test.log",
+            "hosts": self.hostlist_clients,
+            "path": self.workdir,
+            "slots": None,
+            "group": self.server_group,
+            "pool": self.pool,
+            "container": self.container,
+            "processes": self.params.get("np", "/run/ior/client_processes/*"),
+            "ppn": self.params.get("ppn", "/run/ior/client_processes/*"),
+            "intercept": None,
+            "plugin_path": None,
+            "dfuse": None,
+            "display_space": True,
+            "fail_on_warning": False,
+            "namespace": "/run/ior/*",
+            "ior_params": {
+                "oclass": oclass,
+                "flags": self.ior_w_flags,
```

Comment:
Get these values directly from the test yaml:
```suggestion
                "oclass": self.params.get('dfs_oclass', '/run/ior/*'),
                "flags": self.params.get('write_flags', '/run/ior/*'),
```